Lombiq.Analyzers/Lombiq.Analyzers.globalconfig

```diff
@@ -539,6 +541,7 @@ dotnet_diagnostic.MA0101.severity = none
 dotnet_diagnostic.MA0102.severity = none
 dotnet_diagnostic.MA0105.severity = warning
 dotnet_diagnostic.MA0106.severity = none
+dotnet_diagnostic.MA0110.severity = none
```

Comment:
![image](https://github.com/Lombiq/.NET-Analyzers/assets/92299130/59473c8e-1fae-4363-a657-62bdf446627e)
Accoding to MA0009 this code should be compilant
![image](https://github.com/Lombiq/.NET-Analyzers/assets/92299130/7ea9edae-4c76-44c0-b2c0-262e1d9a1645)

But it triggers MA0110. Regex source generation requires the Regex to partial. I would have to then convert the class which was non-partial to partial to be able to use the regex. Do we want that?

Reply:
The example you show is specifically what benefits from source generators from most, since it's `RegexOptions.Compiled`. The line you highlight isn't, this is important.